stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/list/adapter/view/internal/ImageAttachmentsGroupView.kt

```diff
@@ -109,18 +109,13 @@ internal class ImageAttachmentsGroupView : ConstraintLayout {
             if (imageWidth != null && imageHeight != null) {
                 val ratio = (imageWidth / imageHeight).toString()
                 this.setDimensionRatio(imageAttachmentView.id, ratio)
+            } else {
+                constrainHeight(imageAttachmentView.id, LayoutParams.WRAP_CONTENT)
```

Comment:
Useless comment.

Our public bot seems to approve commits to main eagerly so I'm leaving a comment here preventing a merge.